node/src/runner.rs

```diff
@@ -201,6 +205,10 @@ async fn start(
     run_options: &RunOptions,
     chain_config: ChainConfig,
 ) -> Result<()> {
+    if let Some(mock_time) = run_options.mock_time {
+        set_mock_time(*chain_config.chain_type(), mock_time)?;
```

Comment:
Please... no. This isn't necessary. Use RPC for this. Unless I'm missing something?

What's the benefit of setting a constant time value on startup?